WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListFragment.kt

```diff
@@ -183,10 +193,21 @@ class ActivityLogListFragment : Fragment() {
                             viewModel.site,
                             event.activityId,
                             BACKUP_DOWNLOAD_REQUEST_CODE)
-                    // todo: annmarie replace with the ActivityLauncher for showing restore details
-                    is ShowRestore -> displayRewindDialog(event) }
+                    is ShowRestore -> {
```

Comment:
Idea (💡): I wonder if moving the `if/else` logic to the view model would be better? This will help reduce the complexity on the activity level (keep it as dump as possible) and help with testing this logic as well. Wdyt? For example, having `ShowRestoreScreen` and `ShowRestoreDialog` out of `ShowRestore`, which will be then driven by the view model.

PS: Same applies for the `if/else` logic further above on the `showRewindDialog` live data object.
PSS: I know that this logic would be eventually removed at some point, but until them and since this will land on `develop`, maybe it is worth the extra effort.

Reply:
> I know that this logic would be eventually removed at some point

Lol, but great idea. Let me implement and push the commit back. 👍

Reply:
@ParaskP7 in commit e081fe2. Can you give it a whirl

Reply:
Love it @zwarm ! 🌟 

PS: Since now we have additional flows on the view model, mainly creating a couple of test for those flows would be beneficial, wdyt? Just copy-paste the existing tests and add the differentiator factor. This way we would have full test coverage for this feature.